config/jobs-chromeos.yaml

```diff
@@ -551,3 +551,24 @@ jobs:
   tast-ui-arm64-qualcomm: *tast-ui-job
   tast-ui-x86-intel: *tast-ui-job
   tast-ui-x86-amd: *tast-ui-job
+
+  v4l2-decoder-conformance-job: &v4l2-decoder-conformance-job
```

Comment:
@musamaanjum just one more thing: the `v4l2-decoder-conformance-job` anchor should be moved higher up in this file, in the `_anchors:` section.

Reply:
Sure.